### ISSUE TITLE
libcudnn_cnn.so.9.1.0 issue 

### ISSUE THREAD

**Issue by kowshik24** [m-bain/whisperX#903]
I was facing issue while creating docker container i am getting the libcudnn_cnn.so.9.1.0 issue is there any docker container or other options?

**Comment by dmoraine** [comment]
Since recently, I have also been confronted with the same problem when using the library in a Colab environment:

> Unable to load any of {libcudnn_cnn.so.9.1.0, libcudnn_cnn.so.9.1, libcudnn_cnn.so.9, libcudnn_cnn.so}

```
# Load model
model_dir = "./model/"
model = whisperx.load_model("large-v3", "cuda", compute_type="float32", threads=4, download_root=model_dir)

# Load audio
audio = whisperx.load_audio(merged_audio_path)

# Transcribe audio
result = model.transcribe(audio, batch_size=batch_size)  #--> issue
```

**Comment by federicotorrielli** [comment]
See https://github.com/m-bain/whisperX/issues/901 for further info

**Comment by Barabazs** [comment]
Fixed in latest release: [v3.2.0](https://github.com/m-bain/whisperX/releases/tag/v3.2.0)

**Comment by karynaur** [comment]
https://github.com/m-bain/whisperX/issues/902#issuecomment-2646634513 worked! the LD_LIBRARY_PATH was causing the issue. 
i just ran my code with the variable from the venv

```bash
LD_LIBRARY_PATH=venv/lib/python3.10/site-packages/nvidia/cudnn/lib/ CUDA_VISIBLE_DEVICES=2 python3 file.py
```
